src/en/charms-storage.md

```diff
@@ -237,6 +238,53 @@ default values:
 juju deploy postgresql --storage pgdata=ebs,10G cache=ebs-ssd
 ```
 
+### Remove storage
+
+If you remove a unit that has storage, the storage is removed with the unit.
```

Comment:
As I understand it, yes, this is what happens now. I _think_ what @axw was saying above in the "if you remove a unit that has storage, nothing happens _yet_" is that the long-term persistent storage story has not yet been implemented, so the new stuff (storage remaining in the model even through the unit is destroyed) doesn't yet apply.